interlok-core/src/main/java/com/adaptris/core/DefaultSerializableMessageTranslator.java

```diff
@@ -89,6 +90,13 @@ public AdaptrisMessage translate(SerializableMessage message) throws CoreExcepti
         adaptrisMessage = messageFactory.newMessage(message.getContent(), message.getContentEncoding(),
             convertMap(message.getMessageHeaders()));
       }
+
+      if (message.getMessageHeaders().containsKey("_interlokMessageSerialization")) {
+        if (message.getMessageHeaders().get("_interlokMessageSerialization").equals("BASE64")) {
+          adaptrisMessage.setPayload(Base64.getDecoder().decode(message.getContent()));
```

Comment:
Probably need to make a constant out of `_interlokMessageSerilization` into CoreConstants so that you can use it in ServiceTester *easily*. Otherwise too much magic text.

Reply:
I think the if statement could be simplified to be (Base64 should probably be another constant??)

```
if ("BASE64".equalsIgnoreCase(message.getMessageHeaders().get("_interlokMessageSerilization") {
.. do stuff.
}
```